pkg/loghttp/push/otlp.go

```diff
@@ -306,6 +310,36 @@ func otlpToLokiPushRequest(ctx context.Context, ld plog.Logs, userID string, ten
 	return pr
 }
 
+func recordScopeUsage(stats *Stats, resAttrs pcommon.Map, scope pcommon.InstrumentationScope, logs int) {
+	var lang string
+	l, ok := resAttrs.Get(attrSdkLanguage)
+	if ok {
+		lang = l.AsString()
+	} else {
+		lang = unknown
+	}
+	scopeName := scope.Name()
```

Comment:
Isn't the cope name for logs the logger name like `com.mycompany.ecommerce.controller.OrderController`? Note that the scope name for oTel logs is very different from what it is for traces as for traces it's the instrumentation lib.

If so, don't we have PII and cardinality problems using it? 

Examples with a springboot logback app:
<img width="1130" alt="image" src="https://github.com/user-attachments/assets/af92665a-6d87-4587-8eb1-0feda0305537" />

![image](https://github.com/user-attachments/assets/bda1f865-056f-4a27-8c6a-3005b8e39560)

Reply:
you are right - that's why we want to add a feature toggle.

The scope can also be `org.postgres.driver` - which is what we're really interested in.